architecture/core_objects/builds.adoc

```diff
@@ -41,72 +89,70 @@ There are three available link:openshift_model.html#build-strategies[build strat
+The resulting object depends on the builder used to create the image.
 
 [#docker-build]
-== Docker Build
-Docker builds invoke the plain https://docs.docker.com/reference/commandline/cli/#build[docker build] command, and therefore expect a repository with a `Dockerfile` and all required directories.
+=== Docker Build
+Docker builds invoke the plain https://docs.docker.com/reference/commandline/cli/#build[docker build command], and therefore expect a repository with a *_Dockerfile_* and all required directories.
 
 [#sti-build]
-== STI Build
+=== STI Build
 link:../../creating_images/sti.html[Source-to-image (STI)] is a tool for
 building reproducible Docker images. It produces ready-to-run images by
 injecting a user source into a docker image and assembling a new docker image.
 The new image incorporates the base image and built source, and is ready to use
-with the `docker run` command. STI supports incremental builds which re-use
+with the `docker run` command. STI supports incremental builds, which re-use
 previously downloaded dependencies, previously built artifacts, etc.
 
-*What are the goals of STI?*
+The advantages of STI include:
 
 [horizontal]
-Image flexibility:: STI allows you to use almost any existing Docker image as
-the base for your application. STI scripts can be written to layer application
-code onto almost any existing Docker image, so you can take advantage of the
-existing ecosystem. Note that currently STI relies on `tar` and `untar` to
-inject application source so the image needs to be able to process tarred
-content.
-
-Speed:: Adding layers as part of a *_Dockerfile_* can be slow. With STI, the
-assemble process can perform a large number of complex operations without
-creating a new layer at each step. In addition, STI scripts can be written to
-re-use dependencies stored in a previous version of the application image rather
-than re-downloading them each time the build is run.
-
-Patchability:: If an underlying image needs to be patched due to a security
-issue, OpenShift can use STI to rebuild your application on top of the patched
-builder image.
+Image flexibility:: STI scripts can be written to layer application code onto
+almost any existing Docker image, taking advantage of the existing ecosystem.
+Note that, currently, STI relies on `tar` and `untar` to inject application
+source, so the image needs to be able to process tarred content.
+
+Speed:: With STI, the assemble process can perform a large number of complex
+operations without creating a new layer at each step, resulting in a fast
+process. In addition, STI scripts can be written to re-use dependencies stored
+in a previous version of the application image rather than having to download
+them each time the build is run.
+
+Patchability:: If an underlying image needs a patch due to a security issue,
+OpenShift can use STI to rebuild your application on top of the patched builder
```

Comment:
This applies to all builds, not just STI. I would rather say something like "STI allows you to rebuild the application consistently if..."